README.md

```diff
@@ -126,6 +128,8 @@ for i := 0; i < 100; i++ {
 
 // done sending notifications, wait for all responses and shutdown
 queue.Wait()
+// all responses received, wait for all responses to be processed
```

Comment:
Maybe `queue.Wait()` should be renamed to `queue.Close()` -- even though it waits, you still have to wait anyway.